evm_loader/lib/src/tracing/tests/helpers.rs

```diff
@@ -0,0 +1,175 @@
+use crate::account_storage::EmulatorAccountStorage;
+use crate::commands::get_config::{BuildConfigSimulator, ChainInfo, ConfigSimulator};
+use crate::rpc::Rpc;
+use crate::tracing::tracers::openeth::types::to_call_analytics;
+use crate::tracing::tracers::prestate_tracer::tracer::PrestateTracerConfig;
+use crate::tracing::TraceConfig;
+use crate::NeonResult;
+use async_trait::async_trait;
+use ethnum::U256;
+use evm_loader::account::BalanceAccount;
+use evm_loader::solana_program::account_info::AccountInfo;
+use evm_loader::solana_program::clock::{Slot, UnixTimestamp};
+use evm_loader::solana_program::pubkey::Pubkey;
+use evm_loader::solana_program::rent::Rent;
+use evm_loader::types::Address;
+use solana_client::client_error::Result as ClientResult;
+use solana_client::rpc_response::{Response, RpcResponseContext, RpcResult};
+use solana_sdk::account::Account;
+use solana_sdk::commitment_config::CommitmentConfig;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+pub struct TestRpc {
+    accounts: HashMap<Pubkey, Account>,
+}
+
+impl TestRpc {
+    pub fn new(accounts: HashMap<Pubkey, Account>) -> Self {
+        TestRpc { accounts }
+    }
+}
+
+#[async_trait(?Send)]
+impl Rpc for TestRpc {
+    async fn get_account(&self, key: &Pubkey) -> RpcResult<Option<Account>> {
+        Ok(Response {
+            context: RpcResponseContext {
+                slot: 0,
+                api_version: None,
+            },
+            value: self.accounts.get(key).cloned(),
+        })
+    }
+
+    async fn get_account_with_commitment(
+        &self,
+        _key: &Pubkey,
+        _commitment: CommitmentConfig,
+    ) -> RpcResult<Option<Account>> {
+        todo!()
```

Comment:
Will this be implemented later? I understand that it is an implementation for tests, but I think that it is better to use a different macro (such as `unreachable!` or  even a regular panic with a custom message) if we don't need to implement these methods.

Reply:
Replaced with `unreachable!()`.